armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -257,13 +256,21 @@ After=network.target startup-checks.service
 Requires=startup-checks.service
 [Service]
 ExecStart=/usr/bin/bitcoind -daemon -conf=/etc/bitcoin/bitcoin.conf 
+# Note: we need this ExecStartPost command to prepare an .env file
+# in expected format for electrs to have the RPCPASSWORD in its
+# environment. If we contributed a --cookie-file flag to upstream
+# electrs that could parse the .cookie file, we would not need this.
+ExecStartPost=/bin/bash -c " \
+  sleep 5 && \
```

Comment:
maybe increase to 10s or more, just to be safe, as it takes some time until the cookie file is written.

Reply:
Done.